### PR TITLE
<StatisticsWidget /> - improve drivers API

### PR DESCRIPTION
**Pull request by maxvektor**
### 🔦 Summary
Replace exposed Tooltip driver with separated functions.

### CODE REVIEW

**Comment by maxvektor** [comment]
done

File: src/StatisticsWidget/StatisticsWidget.uni.driver.js [hunk 1]
```diff
@@ -33,12 +40,27 @@ export const statisticsWidgetDriverFactory = (base, body) => {
...
+    hideStatisticsInfoTooltip: async index => {
+      const tooltip = await getTooltipDriver(index);
+      await tooltip.mouseLeave();
+    },
+
+    /** Get the text of the info tooltip */
+    getStatisticsInfoTooltipText: async index => {
+      const tooltip = await getTooltipDriver(index);
+
+      if (!(await tooltip.tooltipExists())) {
```
Comment (mosheKerbel) [inline src/StatisticsWidget/StatisticsWidget.uni.driver.js]: add test case - should return null when tooltip does not exist

File: src/StatisticsWidget/test/StatisticsWidget.spec.js [hunk 2]
```diff
@@ -92,13 +94,16 @@ describe('StatisticsWidget', () => {
...
-        await tooltip.mouseEnter();
+        expect(await driver.getStatisticsInfoTooltipText(0)).toBeNull();
+
+        await driver.showStatisticsInfoTooltip(0);
+        expect(await driver.getStatisticsInfoTooltipText(0)).toBe(description);
 
-        expect(await tooltip.tooltipExists()).toBeTruthy();
-        expect(await tooltip.getTooltipText()).toBe(description);
+        await driver.hideStatisticsInfoTooltip(0);
+        await eventually(async () => {
```
Comment (mosheKerbel) [inline src/StatisticsWidget/test/StatisticsWidget.spec.js]: 👍

File: src/StatisticsWidget/test/StatisticsWidget.spec.js [hunk 3]
```diff
@@ -82,23 +83,18 @@ describe('StatisticsWidget', () => {
...
 
         const { driver } = render(<StatisticsWidget {...data} />);
-        const tooltip = await driver.getStatisticInfoTooltip(0);
-
-        expect(await tooltip.exists()).toBeTruthy();
-        await tooltip.mouseEnter();
 
-        expect(await tooltip.tooltipExists()).toBeTruthy();
-        expect(await tooltip.getTooltipText()).toBe(description);
+        expect(await driver.getInfo(0)).toBe(description);
```
Comment (mosheKerbel) [inline src/StatisticsWidget/test/StatisticsWidget.spec.js]: 👍 
plz add another scenario
`should return null when there is no info`

**Comment by mosheKerbel** [comment]
just to align others who may view this PR
all the naming will be modified out side this PR scope across the api + the drivers.

**Comment by mosheKerbel** [comment]
failed due to unrelated flaky tests
merged.
